Cargo.toml

```diff
@@ -40,6 +40,11 @@ ed25519 = ["ed25519-compact"]
 [dev-dependencies]
 enum-iterator = "0.6.0"
 
+# We explicitly lock the version of those transitive dependencies because their
+# Cargo.toml don't parse with the nightly compiler used by Tock. Remove this
+# whole block once CTAP is a library.
+proc-macro2 = "=1.0.43"
```

Comment:
Let's add `# transitive dependency of enum-iterator` even if it's obvious now, it might not be in the future.